rfcs/text/pe/pe.yml

```diff
@@ -0,0 +1,226 @@
+---
+- name: pe
+  title: PE Header
+  group: 2
+  description: These fields contain Windows Portable Executable (PE) metadata.
+  type: group
+  reusable:
+    top_level: false
+    expected:
+      - file
+      - dll
+      - process
+  fields:
+    - name: original_file_name
+      level: extended
+      type: keyword
+      description: Internal name of the file, provided at compile-time.
+      example: MSPAINT.EXE
+
+    - name: file_version
+      level: extended
+      type: keyword
+      short: Process name.
+      description: Internal version of the file, provided at compile-time.
+      example: 6.3.9600.17415
+
+    - name: description
+      level: extended
+      type: keyword
+      description: Internal description of the file, provided at compile-time.
+      example: Paint
+
+    - name: product
+      level: extended
+      type: keyword
+      description: Internal product name of the file, provided at compile-time.
+      example: Microsoft® Windows® Operating System
+
+    - name: company
+      level: extended
+      type: keyword
+      description: Internal company name of the file, provided at compile-time.
+      example: Microsoft Corporation
+
+    - name: imphash
+      level: extended
+      type: keyword
+      short: A hash of the imports in a PE file.
+      description: >
+        A hash of the imports in a PE file. An imphash -- or import hash -- can
+        be used to fingerprint binaries even after recompilation or other code-level
+        transformations have occurred, which would change more traditional hash values.
+
+        Learn more at https://www.fireeye.com/blog/threat-research/2014/01/tracking-malware-import-hashing.html.
+      example: 0c6803c4e922103c4dca5963aad36ddf
+
+    - name: architecture
+      level: extended
+      type: keyword
+      description: CPU architecture target for the file.
+      example: x64
```

Comment:
All of these fields are already in `schemas/pe.yml`, so we can remove them from this file.

This file is either supplemental fields, or we can also do overrides to existing fields when appropriate. But this RFC is only for new fields, so keep the `- fields:` above, and delete all the way to the `architecture` field.